v0/destinations/mp/data/MPIdentifyConfig.json

```diff
@@ -29,6 +29,12 @@
     "required": false,
     "sourceFromGenericMap": true
   },
+  {
+    "destKey": "$ignore_time",
+    "sourceKeys": "ignoreTime",
+    "required": false,
+    "sourceFromGenericMap": true
```

Comment:
We don't have `ignoreTime` in GenericFieldMapping.json the mapping will fail at this point.

Reply:
Segment supports this by setting context.active = false for identify calls. Doing this will set $ignore_time to true for mix panel.
So rather than using the same name `ignoreTime` for source can we use `active`?

Reply:
My idea was, by default set ignoreTime to true. And if the user sends any other value we will accept that.

Reply:
I don't think keeping default true is good idea as during creation this will cause disparity on updated TS also we should update the updated TS always unless user sends this attribute as true